spec/core_functions/color/legacy.hrx

```diff
@@ -0,0 +1,182 @@
+<===> options.yml
+---
+:ignore_for:
+- libsass
+
+<===>
+===================================================================
+<===> rgb/input.scss
+@use "sass:color";
+a {b: color.is-legacy(rgb(0 255 0))}
+
+<===> rgb/output.css
+a {
+  b: true;
+}
+
+<===>
+================================================================================
+<===> hwb/input.scss
+@use "sass:color";
+a {b: color.is-legacy(hwb(0, 50%, 0%))}
+
+<===> hwb/output.css
+a {
+  b: true;
+}
+
+<===>
+================================================================================
+<===> hsl/input.scss
+@use "sass:color";
+a {b: color.is-legacy(hsl(110, 31%, 32%))}
+
+<===> hsl/output.css
+a {
+  b: true;
+}
+
+<===>
+================================================================================
+<===> srgb/input.scss
+@use "sass:color";
+a {b: color.is-legacy(srgb(115, 20, 210))}
+
+<===> srgb/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> srgb-linear/input.scss
+@use "sass:color";
+a {b: color.is-legacy(?????)}
```

Comment:
@mirisuzanne @dvdherron @SondraE and same for here?

a {b: color.is-legacy(color(srgb-linear 0.45098 0.07843 0.823530))}